app/scripts/directives/resources.js

```diff
@@ -1,6 +1,65 @@
 'use strict';
 
 angular.module('openshiftConsole')
+  .directive('containerStatuses', function($filter) {
+    return {
+      restrict: 'E',
+      scope: {
+        pod: '=',
+        startDebugTerminal: '=?',
```

Comment:
I know I suggested this name, but I think maybe `onDebugTerminal` is better :)  sorry